.eslintrc.js

```diff
@@ -6,6 +6,8 @@ module.exports = {
     'require-await': 'error',
     'no-return-await': 'off',
     semi: ['error', 'never'],
-    'comma-dangle': ['error', 'never']
+    'comma-dangle': ['error', 'never'],
```

Comment:
btw this is why you have `comma-dangle=rue` :)

Reply:
i know, but it looks so weird :p